nengo/builder/signal.py

```diff
@@ -211,7 +227,19 @@ def size(self):
     @property
     def strides(self):
         """(tuple) Strides of data in bytes."""
-        return self.initial_value.strides
+        return self.initial_value.strides if self.viewable else None
+
+    @property
+    def type(self):
+        """(str) Type of the Signal ("dense" or "sparse")."""
+
+        return (
+            "sparse" if isinstance(self._initial_value, scipy_sparse.spmatrix)
```

Comment:
`scipy_sparse` could be `None` here, I think, so we have to account for that.

Reply:
This part is removed in the next commit